src/main/java/uk/gov/hmcts/reform/draftstore/controllers/DraftController.java

```diff
@@ -112,7 +112,7 @@ public DraftList readAll(
 
         URI newClaimUri = fromCurrentRequest().path("/{id}").buildAndExpand(id).toUri();
 
-        return created(newClaimUri).build();
+        return withSecretsWarning(secrets, created(newClaimUri));
```

Comment:
I wish java had extension methods...

Perhaps we'll try Kotlin in Spring Boot 2... :)